### ISSUE TITLE
Interface to solving equations

### ISSUE THREAD

**Issue by mikldk** [r-cas/caracas#6]

**Comment by mikldk** [comment]
Suggestions for names? 

* Overload the existing `solve()`
* `equation()`
* `work()`
* `settle()`
* `answer()`
* ~~`find()`~~ is often related to file system
* ...

**Comment by hojsgaard** [comment]
solve()

**Comment by mikldk** [comment]
Currently there are four separate functions:

* `inverse()` finds the inverse of a symbolic matrix
* `linsolve()` solves `Ax = b` by `x = A^{-1} b` using `inverse()`
* `rootsolve()` finds roots in non-linear equations, possible a system
* `systemsolve()` has a `LHS` and `RHS` and uses `rootsolve()` on `LHS - RHS`. 

Maybe we should have one interface function like `solve()`, but it may be difficult for users to understand behavior as a system of equations are represented as a matrix. Then there must be a convention like if `LHS` is a `1xm` matrix then it is non-linear etc.

So currently the above is implemented, but the interface is in no way set in stone.

Related to #9.

**Comment by mikldk** [comment]
Names:
```
solve_lin():
  solve_lin(A)
  solve_lin(A, b)

solve_sys:
  solve_sys(lhs, vars) # rhs == 0
  solve_sys(lhs, rhs, vars)
```

**Comment by mikldk** [comment]
Fixed by 9f31584ad67c57cbd8b5f42e51e24d9a2beb806c.
